lib/middleware/errors.js

```diff
@@ -24,14 +25,18 @@ function _applicationError(err, req, res, next) {
 function _authenticationError(err, req, res, next) {
   logger.log('warn', {'Unauthorized': 'Invalid user credentials'});
   logger.log('warn', {'Error stack': err.stack});
-  res.redirect(uriConstructor.redirectUrl(req.originalUrl));
+
+  return fetch(config.LOGOUT_URL, { headers: req.headers })
```

Comment:
Should we handle the error here too? I mean the request might fail but we still want to get redirected to the login page.
So we can maybe have:
```
return fetch(config.LOGOUT_URL, { headers: req.headers })
    .catch(err => err)// or null, or w/e
    .then(() => {
      res.redirect(uriConstructor.redirectUrl(req));
    });
```